src/test/java/org/broadinstitute/hellbender/utils/hdf5/HDF5PoNCreatorUnitTest.java

```diff
@@ -65,6 +69,36 @@ public void testCreateVariance() {
     }
 
     @Test
+    public void testCreateLargePoN(){
+        // final JavaSparkContext ctx = SparkContextFactory.getTestSparkContext();
+        // Creates a large PoN of junk values.
+        final int numEigenSamples = 20;
```

Comment:
Although this is consistent with the rest of the codebase, `eigensamples` is preferable to `eigenSamples`.